src/selectors/userSession/userSession.js

```diff
@@ -15,7 +15,13 @@ export function loggedIn(state) {
   /* The user may be authenticated by different means, including outside the UI.  Defer
    * to the response from Plone, sepcifically whether Plone presents an option to log
    * in. */
-  return !state.actions.actionsById.user.login;
+
+  if (
+    typeof state.actions.actionsById.user.login === 'undefined' &&
+    typeof state.actions.actionsById.user.logout === 'undefined'
+  )
+    return false;
+  return !!state.actions.actionsById.user?.logout;
```

Comment:
Can you elaborate on why you switched from `!...login` to `!!...logout`?  Maybe even add a comment on that if the reason is something someone else reading the code should know?